types/setting.go

```diff
@@ -815,6 +821,28 @@ var (
 		ReadOnly: false,
 		Default:  CniNetworkNone,
 	}
+
+	SettingDefinitionRecurringSuccessfulJobsHistoryLimit = SettingDefinition{
+		DisplayName: "Cronjob Successful Jobs History Limit",
+		Description: "This setting specifies how many successful backup or snapshot job histories should be retained. \n\n" +
+			"Set the value to 0 means that it won't keep history after the job finished.",
```

Comment:
```suggestion
			"History will not be retained if the value is 0.",
```

same in the below

Reply:
Thank you, I will follow the comment to fix.

https://github.com/longhorn/longhorn-manager/blob/8511cfdf0080dbf35c1dae4e00638ed9b345984b/types/setting.go#L827-L828

https://github.com/longhorn/longhorn-manager/blob/8511cfdf0080dbf35c1dae4e00638ed9b345984b/types/setting.go#L838-L839